cli/tests/pcluster/schemas/test_cluster_schema.py

```diff
@@ -486,7 +487,33 @@ def test_scheduling_schema(mocker, config_dict, failure_message):
                     }
                 ],
             },
-            "A Compute Resource needs to specify either InstanceType or Instances.",
+            "A Compute Resource needs to specify Instances, InstanceType or CapacityReservationId.",
+        ),
+        (
+            {
+                "Name": "Flex-Queue",
+                "Networking": {"SubnetIds": ["subnet-12345678"]},
+                "AllocationStrategy": "lowest-price",
+                "ComputeResources": [
+                    {"Name": "compute_resource1", "CapacityReservationTarget": {"CapacityReservationId": "id"}}
```

Comment:
can you add a case when CapacityReservationTarget/CapacityReservationResourceGroupArn is specified in place of CapacityReservationTarget/CapacityReservationId ?

Reply:
it's already there:

```
{
                "Name": "Flex-Queue",
                "Networking": {"SubnetIds": ["subnet-12345678"]},
                "AllocationStrategy": "lowest-price",
                "ComputeResources": [
                    {
                        "Name": "compute_resource1",
                        "CapacityReservationTarget": {"CapacityReservationResourceGroupArn": "arn"},
                    }
                ],
            },
            "A Compute Resource needs to specify Instances, InstanceType or CapacityReservationId.",
```